pkg/network/ebpf/c/prebuilt/usm.c

```diff
@@ -49,8 +49,22 @@ int BPF_BYPASSABLE_KPROBE(kprobe__tcp_sendmsg, struct sock *sk) {
     return 0;
 }
 
+SEC("raw_tracepoint/net/netif_receive_skb")
+int raw_tracepoint__net__netif_receive_skb(void *ctx) {
+    CHECK_BPF_PROGRAM_BYPASSED()
+    log_debug("tracepoint/net/netif_receive_skb");
+    // flush batch to userspace
+    // because perf events can't be sent from socket filter programs
+    http_batch_flush(ctx);
+    http2_batch_flush(ctx);
+    terminated_http2_batch_flush(ctx);
+    kafka_batch_flush(ctx);
+    postgres_batch_flush(ctx);
+    return 0;
```

Comment:
since that's just a duplication of tracepoint__net__netif_receive_skb__pre_4_17_0 - we should create an helper function to be used instead of manually ensuring both implementations match

Reply:
same comment for `runtime,c`